nixos/modules/services/databases/postgresql.nix

```diff
@@ -181,6 +227,10 @@ in
         local all all              ident
         host  all all 127.0.0.1/32 md5
         host  all all ::1/128      md5
+
+        ${concatMapStrings (user: ''
+          local all ${user.name} peer
+        '') cfg.ensureUsers}
```

Comment:
We already use the (obsolete) `ident` option above, so I think this is reduntant unless you use an idmap.

Reply:
Thanks!